.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 python:
   - "2.7"
 jdk:
-  - oraclejdk8
+  - oraclejdk7
```

Comment:
The original gate used to run on JDK7 only, so I would not be afraid to switch back just to JDK7. In fact all of us are using JDK8 to develop (at least I do), so the JDK8 compatibility is naturally guaranteed. Should something sneak through, it would be discovered by graal-core gate when updating Truffle dependency, as they use oraclejdk8, I assume. These are the thoughts that are influencing my opinion, but in general I am ambivalent:

**Pros**: Increased coverage and better sanity check
**Cons**: Longer execution time